packages/flutter/test/widgets/image_test.dart

```diff
@@ -1199,7 +1199,7 @@ class TestImageProvider extends ImageProvider<TestImageProvider> {
   }
 
   @override
-  ImageStreamCompleter load(TestImageProvider key) => _streamCompleter;
+  ImageStreamCompleter load(TestImageProvider key, DecoderCallback decode) => _streamCompleter;
 
   void complete() {
     _completer.complete(ImageInfo(image: TestImage()));
```

Comment:
would be good to have a test specifically of ResizeImage itself.

Reply:
Added in `image_provider_test.dart`